### PR TITLE
Replace spray with play and get ready for the gui

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

**Comment by 2m** [comment]
Should go together with https://github.com/scala/scala-jenkins-infra/pull/72

**Comment by 2m** [comment]
Review by @adriaanm

File: build.sbt [hunk 1]
```diff
@@ -13,37 +11,35 @@ lazy val deps: Seq[sbt.Def.Setting[_]] =  Seq(
...
-    "io.spray"          %% "spray-httpx"    % "1.3.2",
     "io.spray"          %% "spray-json"     % "1.3.1"
   ))
 
 lazy val amazonDeps: Seq[sbt.Def.Setting[_]] =  Seq(
   libraryDependencies += "com.amazonaws" % "aws-java-sdk" % "1.9.13")
 
+lazy val guiSettings: Seq[sbt.Def.Setting[_]] = Seq(
+  routesGenerator := InjectedRoutesGenerator,
+  javaOptions in Universal += "-Dhttp.port=8888"
```
Comment (adriaanm) [inline build.sbt]: let's drop this and move this into scala-jenkins-infra

Comment (2m) [inline build.sbt]: Removed.

**Comment by adriaanm** [comment]
LGTM! :shipit:

**Pull request merged**